demos/guru_scripts/docker/tutorial/3.x/gsql101/gsql101.gsql

```diff
@@ -0,0 +1,70 @@
+#reset the catalog
+DROP ALL
+
+#create the schema
+CREATE VERTEX person (PRIMARY_ID name STRING, name STRING, age INT, gender STRING, state STRING)
+CREATE UNDIRECTED EDGE friendship (FROM person, TO person, connect_day DATETIME)
+
+CREATE GRAPH social (person, friendship)
+
+#create the loading job
+CREATE LOADING JOB load_social FOR GRAPH social {
+   DEFINE FILENAME file1="/home/tigergraph/tutorial/gsql101/person.csv";
+   DEFINE FILENAME file2="/home/tigergraph/tutorial/gsql101/friendship.csv";
+
+   LOAD file1 TO VERTEX person VALUES ($"name", $"name", $"age", $"gender", $"state") USING header="true", separator=",";
+   LOAD file2 TO EDGE friendship VALUES ($0, $1, $2) USING header="true", separator=",";
+}
+
+#run the loading job
+RUN LOADING JOB load_social
+
+#built-in queries
+Use Graph Social
```

Comment:
Please read GSQL 101, 3.0 doc
https://docs.tigergraph.com/start/gsql-101/built-in-select-queries

We have removed all SELECT count(*) queries. We use REST call instead. Please copy those restcall curl commands here, See the above link.